lib/JSON/Validator/Joi.pm

```diff
@@ -70,6 +73,8 @@ sub props {
     push @{$self->{required}}, $name if $property->{required};
     $self->{properties}{$name} = $property->compile;
   }
+  @{$self->{required}} = sort @{$self->{required}}
+    if $self->{required} && @{$self->{required}};
```

Comment:
I'm not sure why you want to sort the keys. Is it to get the test through?

Reply:
test was failed because` each %%properties`

Reply:
bag... got it )) thanks